next.config.js

```diff
@@ -121,6 +121,7 @@ module.exports = withPlugins([[withBundleAnalyzer], [withSourceMaps]], {
     VERCEL_URL: process.env.VERCEL_URL,
     SITE_IMAGERY_API_URL: SITE_IMAGERY_API_URL,
     WIDGET_URL: process.env.WIDGET_URL,
+    TEST_MFA_URL: process.env.TEST_MFA_URL,
```

Comment:
We do not need the env TEST_MFA_URL for our normal application, but only for testing, so I do not think it is necessary do add it here, do we?

Reply:
only necessary for testing.

Reply:
Instead of this place, TEST_MFA_URL should be added to https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.example.